src/centrality/edge-betweenness.jl

```diff
@@ -0,0 +1,102 @@
+"""
+    edge_betweenness_centrality(g, k)
+
+Compute the [edge betweenness centrality](https://en.wikipedia.org/wiki/Centrality#Betweenness_centrality) of an edge `e`.
+It is defined as the sum of the fraction of all-pairs shortest paths that pass through `e`
+``
+bc(e) =  \\sum_{s, t \\in V}
+\\frac{\\sigma_{st}(e)}{\\sigma_{st}}
+``.
+
+where `V`, is the set of nodes, \\frac{\\sigma_{st}} is the number of shortest-paths, and \\frac{\\sigma_{st}(e)} is the number of those paths passing through edge.
+
+### Optional Arguments
+- `normalize=true`: If true, normalize the betweenness values by the
+total number of possible distinct paths between all pairs in the graphs.
+For an undirected graph, this number is ``\\frac{(|V|-1)(|V|-2)}{2}``
+and for a directed graph, ``{(|V|-1)(|V|-2)}``.
+
+
+### References
+- Brandes 2001 & Brandes 2008
+
+# Examples
+```jldoctest
+julia> using Graphs
+
+julia> edge_betweenness_centrality(star_graph(5))
+Dict{Graphs.SimpleGraphs.SimpleEdge{Int64}, Float64} with 4 entries:
+  Edge 1 => 2 => 0.4
+  Edge 1 => 3 => 0.4
+  Edge 1 => 4 => 0.4
+  Edge 1 => 5 => 0.4
+
+julia> edge_betweenness_centrality(path_digraph(6))
+Dict{Graphs.SimpleGraphs.SimpleEdge{Int64}, Float64} with 5 entries:
+  Edge 4 => 5 => 0.266667
+  Edge 1 => 2 => 0.166667
+  Edge 3 => 4 => 0.3
+  Edge 5 => 6 => 0.166667
+  Edge 2 => 3 => 0.266667
+```
+"""
```

Comment:
We probably should also have an example with `normalize=false`  and maybe (but that might be a bit too much text) one with a different `distmx`.

Reply:
✅